src/widgets/settingspages/GeneralPage.cpp

```diff
@@ -582,6 +582,7 @@ void GeneralPage::initLayout(GeneralPageView &layout)
     layout.addCheckbox("Chatterino", s.showBadgesChatterino);
     layout.addCheckbox("FrankerFaceZ (Bot, FFZ Supporter, FFZ Developer)",
                        s.showBadgesFfz);
+    layout.addCheckbox("Pronouns", s.showBadgesPronouns);
```

Comment:
Other 3rd party badge service settings indicate which service they are from (or badges are named after the service itself), I think it would be good to somehow indicate that `Pronouns` badges are provided by alejo.io